evm/db/chain.py

```diff
@@ -40,6 +40,9 @@
 )
 
 
+CANONICAL_HEAD_HASH_DB_KEY = b'canonical_head_hash'
```

Comment:
What do you think about going ahead and adding a version prefix to *all* of these keys that we generate since we're very likely to iterate on the database schema.